src/main/java/com/iexec/common/docker/client/DockerClientFactory.java

```diff
@@ -26,6 +26,8 @@ public abstract class DockerClientFactory {
 
     private static final Map<String, DockerClientInstance> clientsMap = new HashMap<>();
 
+    private DockerClientFactory() {}
```

Comment:
`@NoArgsConstructor(access = AccessLevel.PRIVATE)`

Reply:
Do you prefer to use lombok annotation for this ?

Reply:
Yes. It reduces boilerplate code.